downloader_app/run_celery_downloader.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+
+import os
+import sys
+
+import pandas as pd
+
+from downloader_app import shapefile_module as shpm
+from downloader_app.tasks import add, download_source
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+WORK_DIR = os.path.join(BASE_DIR, "downloader_app")
+sys.path.insert(0, WORK_DIR)
+
+
+# Set the filepath and load in a shapefile.
+SHP_PATH = os.path.join(
+    WORK_DIR, "shape_region/RJ_Mun97_region/RJ_Mun97_region.shp"
+)
+DOWNLOADFILES_PATH = os.path.join(WORK_DIR, "DownloadedFiles")
+
+
+try:
+    add.delay(3, 5)
+except Exception as e:
+    print(e)
+
+
+if __name__ == "__main__":
+    pass
+
+
+class DownloaderSources:
+    def LandDAAC_v5_day():
+        # Parameters
+        # Extract bounding box from shapefile by uf
+        point1, point2 = shpm.extract_shp_boundingbox(SHP_PATH)
```

Comment:
probably SHP_PATH should be passed as a function argument ... not as a global variable